backend/tests/test_routes.py

```diff
@@ -116,6 +148,83 @@ def test_get_points_two_results(db_data, test_data, client):
     assert users == ["Jerusalem", "Tel Aviv"]
 
 
+def test_get_image_no_id(client):
+    response = client.get("/api/image")
+    assert response.status_code != 200
+
+
+def test_post_image(client):
+    response = client.post("/api/image")
+    assert response.status_code != 201
+    image_name = 'geo_icon.png'
+    image_path = os.path.join(BASE_TESTS_DIR, 'tests_images', image_name)
+    response = client.post("/api/image", files={'image': (image_name, open(image_path, 'rb'), 'image/png')})
+    assert response.status_code == 201
+    response_json = response.json()
+    assert response_json['filename'] == image_name
+    image_id = ObjectId(response_json['id'])
+    response = client.get('/api/image', json={'id': response_json['id']})
+    response.raise_for_status()
+    assert response.content == open(image_path, 'rb').read()
+
+
+def test_create_points_with_images(more_data, client):
+    near_jerusalem = [data for data in more_data
+                 if data['user_id'] == 'near Jerusalem'].pop()
+
+    near_tel_aviv = [data for data in more_data
+                 if data['user_id'] == 'near Tel Aviv'].pop()
+
+    points = sorted([near_jerusalem, near_tel_aviv], key=lambda x: x['user_id'])
+    images_names = os.listdir(os.path.join(BASE_TESTS_DIR, 'tests_images'))
+    images_types = ['image/' + name[name.rindex('.') + 1:] for name in images_names]
+    for i in range(2):
+        image_path = os.path.join(BASE_TESTS_DIR, 'tests_images', images_names[i])
+        response = client.post("/api/image", files={'image': (images_names[i], open(image_path, 'rb'), images_types[i])})
```

Comment:
I'd start here by first asserting the response (as it seems it later fails on missing `id` in this response).